### ISSUE TITLE
Issues with certain title

### ISSUE THREAD

**Issue by rootm0s** [platelminto/parse-torrent-title#8]
Hi,

Having issues parsing the name of these films:
```
Johnny.English.2003.1080p.BluRay.x264-[YTS.AG]
Johnny.English.Reborn.2011.1080p.BRRip.x264  [MovieOW]
Johnny.English.Strikes.Again.2018.1080p.BluRay.x264-[YTS.AM]
```

The result is: 
```
{'year': 2003, 'resolution': '1080p', 'quality': 'BluRay', 'codec': 'x264', 'container': 'mp4', 'language': 'English', 'title': 'Johnny', 'encoder': 'YTS.AG'}
{'year': 2011, 'resolution': '1080p', 'quality': 'BRRip', 'codec': 'x264', 'container': 'mp4', 'language': 'English', 'title': 'Johnny', 'encoder': 'MovieOW', 'excess': 'Reborn'}
{'year': 2018, 'resolution': '1080p', 'quality': 'BluRay', 'codec': 'x264', 'container': 'mp4', 'language': 'English', 'title': 'Johnny', 'encoder': 'YTS.AM', 'excess': 'Strikes.Again'}
```

Any idea how to fix it? 🙏

**Comment by platelminto** [comment]
Hmm definitely related to `language` being handled as a specific case, looking into it now.

EDIT: Actually it isn't, just a wrong match ('BluRay' in a title would also get incorrectly parsed). Now seeing if some rules can be added to fix the majority of these cases, otherwise I'll just add these to the known exceptions.

**Comment by rootm0s** [comment]
> Hmm definitely related to `language` being handled as a specific case, looking into it now.
> 
> EDIT: Actually it isn't, just a wrong match ('BluRay' in a title would also get incorrectly parsed). Now seeing if some rules can be added to fix the majority of these cases, otherwise I'll just add these to the known exceptions.

Thanks for taking the time to look into it!

**Comment by platelminto** [comment]
Should now be fixed by updating to the latest release (1.4)!

**Comment by rootm0s** [comment]
> Should now be fixed by updating to the latest release (1.4)!

thanks mate
